common/dds-model.h

```diff
@@ -0,0 +1,66 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+#pragma once
+
+#include <librealsense2/rs.hpp>
+#include <imgui.h>
+#include <realsense_imgui.h>
+#include <set>
+#include <rsutils/type/ip-address.h>
+#include <rsutils/string/hexdump.h>
```

Comment:
Are all these includes needed here? Move to cpp what's used only by implementation and doesn't need to be declared here